nx.json

```diff
@@ -43,7 +43,10 @@
     "test": {
       "dependsOn": [],
       "inputs": ["default", "^production", "{workspaceRoot}/jest.preset.js"],
-      "cache": true
+      "cache": true,
+      "options": {
+        "passWithNoTests": true
```

Comment:
we used `passWithNoTests` directly within npm alias. this mirrors this setting for `test` target that uses inferred test target creation